es2-persistence/src/main/java/com/netflix/conductor/dao/es/index/ElasticSearchDAO.java

```diff
@@ -259,16 +306,22 @@ public void indexWorkflow(Workflow workflow) {
 	@Override
 	public void indexTask(Task task) {
 		try {
+			long startTime = Instant.now().toEpochMilli();
 			String id = task.getTaskId();
 			TaskSummary summary = new TaskSummary(task);
 			byte[] doc = objectMapper.writeValueAsBytes(summary);
 
 			UpdateRequest req = new UpdateRequest(indexName, TASK_DOC_TYPE, id);
 			req.doc(doc);
 			req.upsert(doc);
-			updateWithRetry(req, "Index task into doc_type of task");
-		} catch (Throwable e) {
-			logger.error("Indexing failed {}", e.getMessage(), e);
+			logger.debug("Indexing task document: {} for workflow: {}" + id, task.getWorkflowInstanceId());
+			indexObject(req, TASK_DOC_TYPE);
```

Comment:
This is because workflows are indexed directly from the async worker queue while tasks are buffered in the in-memory queue and then added to the worker queue.